### PR TITLE
Update PCIUtils package

### PR DESCRIPTION
**Pull request by JL2210**

### CODE REVIEW

File: packages/pciutils.rb [hunk 1]
```diff
@@ -2,35 +2,38 @@
...
   def self.build
-    system "sed -i 's,SBINDIR=\$(PREFIX)/sbin,SBINDIR=#{CREW_PREFIX}/bin,' Makefile"
-    system "make", "PREFIX=#{CREW_PREFIX}", "SHARED=yes", "ZLIB=yes"
+    system 'make',
+           "LIBDIR=#{CREW_LIB_PREFIX}",
+           'SBINDIR=$(PREFIX)/bin',
+           "PREFIX=#{CREW_PREFIX}",
+           'SHARED=yes',
+           'ZLIB=yes',
+           'DNS=yes'
```
Comment (uberhacker) [inline packages/pciutils.rb]: I have a feeling this option will slow down the process.

Comment (JL2210) [reply packages/pciutils.rb]: How so? I'm currently testing it on a Commodore 64 emulator.

Comment (uberhacker) [reply packages/pciutils.rb]: Uh, dns requires a network to resolve ip addresses into domain names.  That takes time.  Not sure why we even need this option in the first place.

Comment (JL2210) [reply packages/pciutils.rb]: It only enables DNS support. It doesn't call DNS unless its needed.

Comment (JL2210) [reply packages/pciutils.rb]: The Commodore 64 emulator runs just fine without any network access.

**Pull request merged**